docs/user/index.rst

```diff
@@ -108,11 +108,14 @@ to help you create fantastic documentation for your project.
    :caption: Reference
    :glob:
 
+   /reference/features
```

Comment:
💯

Reply:
Yes, when we introduced it, we said to start using it as the new index -- but we just postponed the design of the index for next iteration :)

I think the work to move the hosting features in as a description list is great, we can probably do that for other pages, too. But it's best to do in just one PR, otherwise we get Git conflicts galore :)